### DISCUSSION TITLE
percy-capybara 5.0.0 doesn't seem to work with cucumber

### DISCUSSION THREAD

**Discussion by erikaxel** [percy/percy-capybara#176] [Q&A]
## The problem

We are using percy-capybara with Cucumber instead of Rspec and it seems not to work with the new version, we get the following error after updating according to the readme:
```
undefined method `snapshot' for #<Capybara::Session> (NoMethodError)
```

It used to work with older versions.

## Environment

- Node version: v12.21.0
- `@percy/cli` version:  percy/cli@^1.0.0-beta.62
- Version of Percy SDK you’re using: 5.0.0

## Details

We tried to use percy-selenium but that didnt work due to versioning issues, it requires a too high version of selenium to work. It might be that relaxing that and using that would be the right solution forward, but we think it would be nice if percy-capybara supports cucumber out of the box.

Create any simple test with cucumber to recreate the issue.

**Comment by Robdel12** [comment]
Hey @erikaxel! Thanks for the report, I'm curious to know what driver you're using in your Capybara Cucumber tests. Also curious how you're invoking Percy in the cucumber tests. 

I'd expect this to work, given we're adding the `percy_snapshot` method to the `page` class: https://github.com/percy/percy-capybara/blob/master/lib/percy/capybara.rb#L113 I also don't spot anything obvious that would break Percy in `cucumber.rb`: https://github.com/teamcapybara/capybara/blob/master/lib/capybara/cucumber.rb

Curious to hear more (and then likely build a reproduction)

**Reply by erikaxel** [reply]
Hi,

Thanks for the response, here is the setup for the driver we are using, I hope this is helpful.
```ruby
def setup_headless_chrome
  client = Selenium::WebDriver::Remote::Http::Default.new
  client.read_timeout = 150

  Capybara.register_driver :headless_chrome do |app|
    capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
      'goog:chromeOptions': { args: %w[headless disable-gpu no-sandbox window-size=1024,768 disable-dev-shm-usage] }
    )

    Capybara::Selenium::Driver.new app,
                                   browser: :chrome,
                                   desired_capabilities: capabilities,
                                   http_client: client
  end

  Capybara.server_port = 9887 + ENV['TEST_ENV_NUMBER'].to_i

  Capybara.javascript_driver = :headless_chrome
end
```
